docker/development/debian/Dockerfile

```diff
@@ -89,7 +89,7 @@ RUN mkdir -p /tmp/open-space-toolkit-core \
 
 ## Open Space Toolkit ▸ I/O
 
-ARG OSTK_IO_VERSION=0.4.3
+ARG OSTK_IO_VERSION=0.4.4
```

Comment:
update to latest the versions of dependencies:
- CORE: `0.4.5`
- IO: `0.4.5`
- MATH: `0.4.6`
- PHYSICS: `0.5.16` (soon to be)

Reply:
fixed